configs/lc/toss/clang.cmake

```diff
@@ -1,3 +1,7 @@
-set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-11.0.1/bin/clang" CACHE FILEPATH "Path to C compiler")
```

Comment:
No CMAKE_C_COMPILER anymore?

Reply:
CARE just uses C++, but I just realized that Umpire has some C code, and this config is used to drive building the dependencies as well as CARE in some cases. So I added the C compiler back in.